modules/jaeger-product-overview.adoc

```diff
@@ -4,7 +4,7 @@ This CONCEPT module included in the following assemblies:
 -rhbjaeger-architecture.adoc
 ////
 
-[id="ossm-jaeger-overview_{context}"]
+[id="jaeger-product-overview_{context}"]
 = Jaeger overview
 
 Jaeger lets service owners instrument their services to get insights into what their architecture is doing.
```

Comment:
There are three sentences there in a row that start with Jaeger. 

What about 

s/Jaeger lets service owners instrument their services to get insights into what their architecture is doing./As a service owner, you can use Jaeger to instrument your services to gather insights into your service architecture./

And maybe that third sentence Jaeger is based on vendor neutral... Can that go after the list? I think it's less important than the first two sentences or the list.